cms/djangoapps/contentstore/views/component.py

```diff
@@ -62,7 +62,7 @@
     "editor-mode-button", "upload-dialog", "image-modal",
     "add-xblock-component", "add-xblock-component-button", "add-xblock-component-menu",
     "add-xblock-component-menu-problem", "xblock-string-field-editor", "publish-xblock", "publish-history",
-    "unit-outline", "container-message"
+    "unit-outline", "container-message", "license-selector"
```

Comment:
If you add a trailing comma, you might not have to edit this line the next time :)